app/javascript/components/verifiable.html

```diff
@@ -1,5 +1,14 @@
 <span>
-  Does <a target="_blank" rel="noopener nofollow" class="external free" :href="page.reference_url">{{ page.reference_url }}</a> provide evidence for this position? –
-  <button v-on:click="verifyStatement()" class="mw-ui-button mw-ui-progressive">Yes, verify</button>
-  <button v-on:click="unverifyStatement()" class="mw-ui-button">No evidence</button>
+  <p>Does <a target="_blank" rel="noopener nofollow" class="external free" :href="page.reference_url">{{ page.reference_url }}</a> provide evidence for this position?</p>
+
+  <span v-if="!nameChanged">
+    <button v-on:click="verifyStatement()" class="mw-ui-button mw-ui-progressive">Yes, verify</button>
+    <button v-on:click="changeName()" class="mw-ui-button">No, name isn't quite right</button>
```

Comment:
It's a tiny thing, but I think this one would be better phrased as "Yes, but their name isn't quite right"